qa/vagrant/build.gradle

```diff
@@ -118,7 +97,7 @@ for (Project subproj : project.rootProject.subprojects) {
 }
 plugins = plugins.toSorted()
 
-copyPackagingArchives {
+tasks.named('copyDistributions') {
   from configurations.allPlugins
```

Comment:
Can we move the creation of this configuration, as well as adding it to this task into the plugin? Only thing we should need in the build script is the part that actually _adds_ dependencies to the configuration. The plugin can ensure it exists and is wired up appropriately.

Reply:
I have it moved in https://github.com/elastic/elasticsearch/pull/45360 which is now based on this PR.